src/main/kotlin/com/devh/vitstore/model/VerificationToken.kt

```diff
@@ -0,0 +1,31 @@
+package com.devh.vitstore.model
+
+import java.sql.Timestamp
+import java.util.*
+import javax.persistence.*
+
+
+@Entity
+@Table(name = "verification_tokens")
```

Comment:
bảng này e ko dùng à? `oauth_tokens`
cái active token gen rồi lưu vào cột này chứ e? `active_token` hay e đang làm kiểu nào đấy?

Reply:
bảng oauth_tokens dùng cho login mà anh. như em comment lúc TO là giờ em gộp lại nè hihi